test/unit/solver/test_jit_rosenbrock.cpp

```diff
@@ -0,0 +1,133 @@
+  for (auto& elem : jacobian.AsVector())
+    elem = 100.0;
+  for (std::size_t i_cell = 0; i_cell < number_of_grid_cells; ++i_cell)
+  {
+    jacobian[i_cell][0][0] = 12.2;
+    jacobian[i_cell][0][1] = 24.3 * (i_cell + 2);
+    jacobian[i_cell][0][2] = 42.3;
+    jacobian[i_cell][1][0] = 0.43;
+    jacobian[i_cell][1][1] = 23.4;
+    jacobian[i_cell][1][2] = 83.4 / (i_cell + 3);
+    jacobian[i_cell][2][0] = 4.74;
+    jacobian[i_cell][2][2] = 6.91;
+    jacobian[i_cell][3][1] = 59.1;
+    jacobian[i_cell][3][3] = 83.4;
+    jacobian[i_cell][4][0] = 78.5;
+    jacobian[i_cell][4][2] = 53.6;
+    jacobian[i_cell][4][4] = 1.0;
+  }
+  solver.AlphaMinusJacobian(jacobian, 42.042);
+  for (std::size_t i_cell = 0; i_cell < number_of_grid_cells; ++i_cell)
+  {
+    EXPECT_NEAR(jacobian[i_cell][0][0], 42.042 - 12.2, 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][0][1], -24.3 * (i_cell + 2), 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][0][2], -42.3, 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][1][0], -0.43, 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][1][1], 42.042 - 23.4, 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][1][2], -83.4 / (i_cell + 3), 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][2][0], -4.74, 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][2][2], 42.042 - 6.91, 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][3][1], -59.1, 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][3][3], 42.042 - 83.4, 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][4][0], -78.5, 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][4][2], -53.6, 1.0e-5);
+    EXPECT_NEAR(jacobian[i_cell][4][4], 42.042 - 1.0, 1.0e-5);
+  }
+}
+
+template<class T>
+using Group1VectorMatrix = micm::VectorMatrix<T, 1>;
+template<class T>
+using Group2VectorMatrix = micm::VectorMatrix<T, 2>;
+template<class T>
+using Group3VectorMatrix = micm::VectorMatrix<T, 3>;
+template<class T>
+using Group4VectorMatrix = micm::VectorMatrix<T, 4>;
+
+template<class T>
+using Group1SparseVectorMatrix = micm::SparseMatrix<T, micm::SparseMatrixVectorOrdering<1>>;
+template<class T>
+using Group2SparseVectorMatrix = micm::SparseMatrix<T, micm::SparseMatrixVectorOrdering<2>>;
+template<class T>
+using Group3SparseVectorMatrix = micm::SparseMatrix<T, micm::SparseMatrixVectorOrdering<3>>;
+template<class T>
+using Group4SparseVectorMatrix = micm::SparseMatrix<T, micm::SparseMatrixVectorOrdering<4>>;
+
+TEST(JitRosenbrockSolver, DenseAlphaMinusJacobian)
+{
+  auto jit{ micm::JitCompiler::create() };
+  if (auto err = jit.takeError())
+  {
+    llvm::logAllUnhandledErrors(std::move(err), llvm::errs(), "[JIT Error]");
+    EXPECT_TRUE(false);
+  }
+  testAlphaMinusJacobian<Group1VectorMatrix, Group1SparseVectorMatrix>(jit.get(), 1);
+  // testAlphaMinusJacobian<Group2VectorMatrix, Group2SparseVectorMatrix>(jit.get(), 4);
+  // testAlphaMinusJacobian<Group3VectorMatrix, Group3SparseVectorMatrix>(jit.get(), 3);
+  // testAlphaMinusJacobian<Group4VectorMatrix, Group4SparseVectorMatrix>(jit.get(), 2);
```

Comment:
```suggestion
  testAlphaMinusJacobian<Group2VectorMatrix, Group2SparseVectorMatrix>(jit.get(), 2);
  testAlphaMinusJacobian<Group3VectorMatrix, Group3SparseVectorMatrix>(jit.get(), 3);
  testAlphaMinusJacobian<Group4VectorMatrix, Group4SparseVectorMatrix>(jit.get(), 4);
```

Reply:
Uncommenting lead to an error for duplicate symbol definition of the function name, at least on my computer using clang and llvm 16.0.6. I added a timestamp to the function name and things works